.pre-commit-config.yaml

```diff
@@ -104,13 +104,21 @@ repos:
           - --comment-style
           - <!--|   ~|  -->
       - id: insert-license
-        files: (\.scss$|\.css$)
+        files: \.css
         args:
           - --use-current-year
           - --license-filepath
           - NOTICE.txt
           - --comment-style
           - /*| *| */
+      - id: insert-license
+        files: \.scss
+        args:
+          - --use-current-year
+          - --license-filepath
+          - NOTICE.txt
+          - --comment-style
+          - /*!| *| */
```

Comment:
I don't understand the need for changing the hook config. What is the purpose for changing the comment style for scss files? What problem does it solve?

Reply:
I changed that because when I use copyright auto update from Intellij it append copyright with format that I specify in the .pre-commit-config.yaml which is described in sass documentation that is comment that exists in the file after transpiled to css.

```
// This comment won't be included in the CSS.

/* But this comment will, except in compressed mode. */

/* It can also contain interpolation:
 * 1 + 1 = #{1 + 1} */

/*! This comment will be included even in compressed mode. */
```